src/pages/SignUp.jsx

```diff
@@ -1,11 +1,109 @@
 import React from "react";
 
-function SignUp() {
+import Email from "../assets/Email.svg";
```

Comment:
Can we use some validation here like Formik or react-hook-form? For email and password!

Reply:
Sure